packages/core/src/button/button.element.scss

```diff
@@ -21,19 +21,6 @@
   & > span > span.button-content {
     @include line-height-eraser(0em);
   }
-
-  .button-icon + .button-content {
-    margin-left: #{$cds-token-space-size-4};
-  }
-
-  .button-content + .button-badge {
-    margin-left: #{$cds-token-space-size-4};
-  }
-
-  .button-icon,
-  .button-badge {
```

Comment:
I wasn't sure about removing this but when I visually tested it (screenshots + line measurements in sketch) there was no visible effect,